src/targets/java/unirest.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+var util = require('util');
+
+module.exports = function(options) {
+	var opts = util._extend({
+		indent : '			'
+	}, options);
+
+	var code = [];
+
+	code.push(util.format(
+			'HttpResponse<JsonNode> jsonResponse = Unirest.%s("%s")', this.source.method.toLowerCase(),
+			this.source.fullUrl));
+	
+	// construct headers
+	if (this.source.headers && this.source.headers.length) {
```

Comment:
you should just use `source.allHeader` that way you don't have to do cookies below.